src/Cli/dotnet/commands/dotnet-sln/list/Program.cs

```diff
@@ -24,10 +25,10 @@ public ListProjectsInSolutionCommand(
 
         public override int Execute()
         {
-            string solutionFileFullPath = SlnCommandParser.GetSlnFileFullPath(_fileOrDirectory);
+            string solutionFileFullPath = SlnFileFactory.GetSolutionFileFullPath(_fileOrDirectory, true);
```

Comment:
nit: especially for bools, prefer specifying what exactly 'true' means:
```suggestion
            string solutionFileFullPath = SlnFileFactory.GetSolutionFileFullPath(_fileOrDirectory, includeSolutionFilterFiles: true);
```

This is especially true if there are multiple boolean parameters at the end.

Reply:
What would you think of having an enum instead with values such as:
`SlnFileFactoryInclude.LegacySolutions` (.sln)
`SlnFileFactoryInclude.Solutions` (.sln and .slnx)
`SlnFileFactoryInclude,SolutionsAndFilteredSolutions` (.sln, .slnx, and .slnf)